packages/badge/index.ts

```diff
@@ -1,3 +1,3 @@
-export { default as Badge } from "./components/badge";
-export { default as BadgeButton } from "./components/badgeButton";
+export { default as Badge } from "./components/Badge";
+export { default as BadgeButton } from "./components/BadgeButton";
```

Comment:
did you mean to capitalize these? the files themselves are still lowercase

Reply:
I had capitalized these files and something in the ui-kit appears to be reverting that change, digging into why that may be happening.

Reply:
think you may need to abandon this change. These files are still lower case